src/main/scala/org/ergoplatform/modifiers/history/HistoryModifierSerializer.scala

```diff
@@ -13,6 +13,8 @@ object HistoryModifierSerializer extends Serializer[ErgoPersistentModifier] {
       ADProofs.modifierTypeId +: ADProofSerializer.toBytes(m)
     case m: BlockTransactions =>
       BlockTransactions.modifierTypeId +: BlockTransactionsSerializer.toBytes(m)
+    case m: Extension =>
+      Extension.modifierTypeId +: ExtensionSerializer.toBytes(m)
```

Comment:
In some future (not in this pull request) we can use `sigmastate.utils.ByteWriter` to serialize Modifiers, this will reduce garbage generated when allocating and copying multiple byte arrays.